rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -24,6 +24,7 @@ futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 http-body = "0.4"
 hyper = { version = "0.14", features = ["server", "http1", "http2", "tcp", "stream"] }
+lambda_http = { git = "https://github.com/awslabs/aws-lambda-rust-runtime" }
```

Comment:
A release isn't cut yet on this crate. This PR necessitates the latest changes from https://github.com/awslabs/aws-lambda-rust-runtime/pull/491